src/plugins/telemetry/server/fetcher.ts

```diff
@@ -152,22 +224,23 @@ export class FetcherTask {
       failureCount,
       failureVersion,
       currentVersion: currentKibanaVersion,
-      lastReported: telemetrySavedObject ? telemetrySavedObject.lastReported : void 0,
+      lastReported,
     };
   }
 
   private async updateLastReported() {
-    this.lastReported = Date.now();
+    this.lastReported$.next(Date.now());
     updateTelemetrySavedObject(this.internalRepository!, {
       reportFailureCount: 0,
-      lastReported: this.lastReported,
+      lastReported: this.lastReported$.value,
     }).catch((err) => {
       err.message = `Failed to update the telemetry saved object: ${err.message}`;
       this.logger.debug(err);
     });
   }
 
   private async updateReportFailure({ failureCount }: { failureCount: number }) {
+    this.isOnline$.next(false);
```

Comment:
Setting as offline whenever we report a failure, so we'll check again in the range of 12h